ngwmn/views.py

```diff
@@ -0,0 +1,54 @@
+"""
+NGWMN UI application views
+"""
+from urllib.parse import urljoin
+
+from flask import abort, render_template
+import requests as r
+
+from . import app
+from .utils import parse_xml, get_well_lithography
+
+
+SERVICE_ROOT = app.config.get('SERVICE_ROOT')
+
+
+@app.route('/well-location/<agency_cd>/<location_id>', methods=['GET'])
+def well_page(agency_cd, location_id):
+    """
+    Well location view.
+
+    :param str agency_cd: agency code for the agency that manages the location
+    :param location_id: the location's identifier
+
+    """
+    root = get_well_lithography(SERVICE_ROOT, agency_cd, location_id)
+    if root is not None and 'gml' in root.nsmap.keys():
+        geolocation_element = root.find('.//gml:Point/gml:pos', namespaces=root.nsmap)
+        geolocation = geolocation_element.text
+        latitude, longitude = geolocation.split(' ')
+        lat_lower = float(latitude) - 0.01
```

Comment:
I bet this won't be the last time we need a bbox parameter. This could be a utils function.

Reply:
Sounds good.